src/content/components/MyBugs/MyBugs.js

```diff
@@ -31,9 +31,12 @@ export class MyBugs extends React.PureComponent {
       showSettings: false,
       emailWasChanged: false,
     };
+    this.onPrefChange = this.onPrefChange.bind(this);
     this.onEmailChange = this.onEmailChange.bind(this);
     this.onEmailSubmit = this.onEmailSubmit.bind(this);
     this.toggleSettings = this.toggleSettings.bind(this);
+
+    prefs.on("bugzilla_email", this.onPrefChange);
```

Comment:
maybe this should be done in componentDidMount and removed in componentDidUnmount, though I didn't notice any errors doing it this way. with the priority guide I think you can do it the same way since I *think* it never gets unmounted